projects/packages/my-jetpack/_inc/components/product-cards-section/protect-card/use-protect-tooltip-copy.ts

```diff
@@ -49,13 +54,26 @@ export function useProtectTooltipCopy(): TooltipContent {
 	const pluginsCount = fromScanPlugins.length || Object.keys( plugins ).length;
 	const themesCount = fromScanThemes.length || Object.keys( themes ).length;
 
+	const criticalThreatCount: number = useMemo( () => {
+		return threats.length
+			? threats.reduce( ( accum, threat ) => ( threat.severity >= 5 ? ( accum += 1 ) : accum ), 0 )
+			: 0;
+	}, [ threats ] );
+
 	const settingsLink = useMemo( () => {
 		if ( isProtectPluginActive ) {
 			return 'admin.php?page=jetpack-protect#/firewall';
 		}
 		return isJetpackPluginActive() ? 'admin.php?page=jetpack#/settings' : null;
 	}, [ isProtectPluginActive ] );
 
+	const protectDashboardUrl = useMemo( () => {
+		if ( isProtectPluginActive ) {
+			return protectPluginDashboardUrl;
+		}
+		return `https://cloud.jetpack.com/scan/${ siteSuffix }`;
```

Comment:
This is just an idea and I'm not sure how much better it would be, but it might be useful to have a `getJetpackCloudLink` utility function that could be used like
```
return getJetpackCloudLink( `/scan/${sitesuffix}` );
```
Let me know what you think 😄

Reply:
Actually I refactored this..
Instead, I refactored class `Protect::get_manage_url()` function to return either the Protect plugin dashboard url or the Jetpack Cloud Scan url based on whether the standalone or Jetpack plugin is installed & activated.
The I pull and use that url here in `use-protect-tooltip-copy.ts`.  I think this would be the more appropriate solution. What do you think?